pkgs/applications/audio/mopidy/listenbrainz.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  pythonPackages,
+  fetchFromGitHub,
+  mopidy,
+}:
+
+pythonPackages.buildPythonApplication rec {
+  pname = "mopidy-listenbrainz";
+  version = "0.3.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner= "suaviloquence";
+    repo = "mopidy-listenbrainz";
+    rev = "v0.3.0";
```

```suggestion
    tag = "v${version}";
```
That's mostly what rec is used for